CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 ## 1.2.0-beta.2 (Unreleased)
 
 - Add `az_iot_provisioning_client_get_request_payload()` to create MQTT payload bodies during Device Provisioning.
+- This version provides new APIs to follow the IoT Plug and Play convention to implement Telemetry, Commands, Properties and Components defined in a DTDL model.
+  - To read/write properties the SDK now provides functions to produce the right payload for components, as shown in the header sdk/inc/azure/iot/az_iot_hub_client_properties.h.
+  - To send telemetry messages, the required header is added to identify components.
+  - When responding to a command invocation the component name is automatically parsed and provided when available.
+  - All new samples follow the IoT Plug and Play convention and can be connected to IoT Hub (with or without DPS), or IoT Central.
```

Comment:
What's DPS?

Reply:
Device Provisioning Service (https://docs.microsoft.com/en-us/azure/iot-dps/).  If you want to sneak in the acrynom expansion into #1899 that'd work for me too.  Thanks

Reply:
Seemed folks would have this domain understanding if they are using the service with the SDK.